packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -249,7 +251,9 @@ export function usePress(props: PressHookProps): PressResult {
 
     let onKeyUp = (e: KeyboardEvent) => {
       if (state.isPressed && isValidKeyboardEvent(e)) {
-        e.preventDefault();
+        if (isSubmitEvent(e)) { // spacebar trigger form submit on key up
```

Comment:
I think this should be
```suggestion
        if (!isSubmitEvent(e)) { // spacebar trigger form submit on key up
```
Currently hitting space on the submit button doesn't work